code/datums/components/storage/food_storage.dm

```diff
@@ -58,6 +58,9 @@
 		to_chat(user, "<span class='warning'>There's something in \the [parent].</span>")
 		return
 
+	if(istype(inserted_item, /obj/item/pen) && user.a_intent == INTENT_HELP || iscyborg(user)) //Allows renaming and prevents borgs from breaking their modules by hiding them inside foods
```

Comment:
Split these into two checks, one for checking pens and the other for checking cyborgs.